mainframe/settings.py

```diff
@@ -24,7 +24,7 @@
 # See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('SECRET_KEY', '')
+SECRET_KEY = os.environ.get('SECRET_KEY', 'SECRET_KEY')
```

Comment:
Should I change second arg back to an empty string?

Reply:
nope, let's keep it like this